CHANGELOG.md

```diff
@@ -1,3 +1,134 @@
+# 1.6.10-beta01 (April 2024)
+
+_Changes since 1.6.2_
+
+## Highlights
+
+- Experimental multiplatform support of `Lifecycle` and `ViewModel`.
+  - The `LifecycleOwner` interface is implemented on the level of Compose Multiplatform itself, providing a `LifecycleOwner` entity for all common composables
+  - The `ViewModel` class can be used on any platform, but it needs a factory for platforms where the necessary type cannot be discovered by class reflection (iOS and Web)
+- Experimental multiplatform support of Jetpack Navigation.
+  - To see the Compose Multiplatform navigation library in action, check out the [nav_cupcake project](https://github.com/MatkovIvan/nav_cupcake) which was converted from the [Navigate between screens with Compose](https://developer.android.com/codelabs/basic-android-kotlin-compose-navigation#2) Android codelab.
+- [Support multi-module projects and libraries publication with Compose resources](https://github.com/JetBrains/compose-multiplatform/pull/4454) <sub>_(Kotlin `2.0.0-Beta05` or higher is required)_</sub>
+
+## Known issues
+
+- `inline fun <reified VM> viewModel(...)` is not available from common due to [compiler bug](https://github.com/JetBrains/compose-multiplatform/issues/3147). Please use `fun <VM> viewModel(KClass, ...)` overload instead.
+- Compose Multiplatform doesn't provide default `ViewModelStoreOwner` yet. For using `ViewModel`s outside of `NavHost` you need to provide custom store owner via `LocalViewModelStoreOwner`. Due to dependencies stability, it's expected to be fixed only in `1.7.0`.
+
+## Features
+
+### Common
```

Comment:
```suggestion
### iOS, Desktop, Web
```
As discussed, we should include only fixes that affect Android here.

Reply:
May be  
```suggestion
### Multiplatform
```
?

Reply:
`Multiplatform` has the same meaning that includes Android